runtime/bindings/python/tests/io_test.py

```diff
@@ -0,0 +1,192 @@
+# Copyright 2023 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import array
+import logging
+import numpy as np
+from pathlib import Path
+import unittest
+
+import iree.compiler
+import iree.runtime as rt
+
+
+MM_TEST_COMPILED = None
+MM_TEST_ASM = r"""
+  #map = affine_map<(d0, d1) -> (d0, d1)>
+  #map1 = affine_map<(d0, d1) -> (d1, d0)>
+  #map2 = affine_map<(d0, d1) -> (d1)>
```

Comment:
How relevant to the test are these affine maps and specific linalg ops? Could elementwise arith ops be used instead?

(using linalg ops is a good way to push the compiler to use HAL/device dispatches instead of just the VM, if that's what you're going for here)

Reply:
There is an RFC to remove elementwise arith ops. I don't want to add more dependencies on it.

This is a really basic matmul+bias. I'm ok with it.

Reply:
(I was just going for not depending on optional or deprecated features while also doing "real math". In truth, I just copied and pasted this from some Turbine test case output)

Reply:
"affine map" is not spoken. It is more of an emotion made manifest.